airbyte-integrations/connectors/source-jdbc/src/main/resources/spec.json

```
@@ -18,9 +18,9 @@
         "type": "string",
         "airbyte_secret": true
       },
-      "jdbc_url": {
+      "connection_properties": {
```

roll back this change to
```suggestion
      "jdbc_url": {
```